__tests__/test-data.test.js

```diff
@@ -50,3 +48,35 @@ describe("api/topics", () => {
     );
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("returns article api ", () => {
+    return request(app).get("/api/articles").expect(200);
```

Comment:
This endpoint isn't the one being tested. This test does not match up with the ticket.

Reply:
^ Agree with Verity, ensure that your describe block reflects '/api/articles' rather than '/api/articles/:article_id'. These are two different endpoints so should be in two different test blocks

Reply:
This test still hasn't changed and is not appropriate to this end point, you're not testing all the articles yet.